brownie-config.yaml

```diff
@@ -1,3 +1,5 @@
+dotenv: .env
```

Comment:
what did you need `.env` for? can you add a `.env.example`?

Reply:
needed it for contract verification in goerli. added in 301da52c1511b7bd4233f4f74ed686ded8fd19b1